### DISCUSSION TITLE
Does not work example on page 'Response Model'

### DISCUSSION THREAD

**Discussion by manlix** [fastapi/fastapi#8896] [Questions] [answered]
### Issue description:

I have installed `FastAPI` in standart way as described in chapter [Installation](https://fastapi.tiangolo.com/#installation):

`
$ pip install fastapi
`

But later got error running code example from chapter [Response Model / Return the same input data](https://fastapi.tiangolo.com/tutorial/response-model/#return-the-same-input-data). 

### Expected result:

Code works properly.

### Actual result:

Code does not work.
Got error:

```
ImportError: email-validator is not installed, run `pip install pydantic[email]`
```
### Preconditions:

* Python: **3.9.5**
* FastAPI: **0.65.2**

### How to reproduce:

Run example code from chapter [Response Model / Return the same input data](https://fastapi.tiangolo.com/tutorial/response-model/#return-the-same-input-data).
Copy code to `main.py`:

```python3
from typing import Optional

from fastapi import FastAPI
from pydantic import BaseModel, EmailStr

app = FastAPI()


class UserIn(BaseModel):
    username: str
    password: str
    email: EmailStr
    full_name: Optional[str] = None


# Don't do this in production!
@app.post("/user/", response_model=UserIn)
async def create_user(user: UserIn):
    return user
```

After that run FastAPI:

```bash
$ uvicorn main:app
```

### The cause of the problem:

Used type `EmailStr` requires installed package `email-validator`.

### Solution:

There is small fix in PR #3367 for docs.

**Comment by YuriiMotov** [comment, marked as answer]
There is a remark about this in current version of Docs.

`pip install email-validator`

or

`pip install "fastapi[standard]"`
